src/Tests/dotnet-new.Tests/Approvals/DotnetCSharpClassTemplatesTest.record.langVersion=9.0.verified/record/TestItem1.cs

```diff
@@ -1,6 +1,8 @@
 ﻿namespace ClassLib
 {
-    public record TestItem1
+    //Record was added in C# 9 and later, so Class was used instead. 
+    //See more info: https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/record
+    public class TestItem1
```

Comment:
I'm confused here as the name of the test says it is testing records on C# 9:
`DotnetCSharpClassTemplatesTest.record.langVersion=9.0.verified/record/TestItem1.cs`

Reply:
I have pushed a change for this [67d76cc](https://github.com/dotnet/sdk/pull/34996/commits/67d76cc706074a949869952cc8441569d3b83ed7)